core/lib/is-idle-phase.js

```diff
@@ -6,7 +6,8 @@ function isIdlePhase(phase) {
   return (
     (phase.arrivalRate === 0 && !phase.rampTo) ||
     phase.arrivalCount === 0 ||
-    phase.maxVusers === 0
+    phase.maxVusers === 0 ||
+    phase.pause > 0
```

Comment:
this means that now the expected amount of workers that we wait for when collecting stats is actually the ones that _are_ running.

We still need the added `emit('stats',..)` for cases were the work is divided among n workers but the last phase is an arrivalCount (which means only one active worker).

Added a test that checks we are dividing properly in scritps that mix `arrivalRate` and `arrivalCount`